code/modules/mob/living/simple_animal/hostile/true_changeling.dm

```diff
@@ -0,0 +1,220 @@
+#define TRUE_CHANGELING_REFORM_THRESHOLD 0 //Can turn back at will, by default
+#define TRUE_CHANGELING_PASSIVE_HEAL 3 //Amount of brute damage restored per tick
+#define TRUE_CHANGELING_FORCED_REFORM 180 //3 minutes
+
+//Changelings in their true form.
+//Massive health and damage, but all of their chems and it's really obvious it's >them
+
+/mob/living/simple_animal/hostile/true_changeling
+	name = "horror"
+	real_name = "horror"
+	desc = "Holy shit, what the fuck is that thing?!"
+	speak_emote = list("says with one of its faces")
+	emote_hear = list("says with one of its faces")
+	icon = 'icons/mob/changeling.dmi'
+	icon_state = "horror1"
+	icon_living = "horror1"
+	icon_dead = "horror_dead"
+	speed = -0.5
+	gender = NEUTER
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = 0
```

Comment:
should be inherited, no?

Reply:
is not, i don't think

Reply:
i can test!

Reply:
you're correct, simple animals have CANPUSH

Reply:
fistbump me bro